build/Dockerfile.ci

```diff
@@ -8,7 +8,8 @@ LABEL stage=build
 WORKDIR /build/
 
 # dos2unix is needed to build CNI plugins
-RUN yum install -y dos2unix
+# jq is needed in e2e test script
```

Comment:
Can you state specifically what it is needed for?

Reply:
Addressed in [e755824](https://github.com/openshift/windows-machine-config-operator/commit/e755824ec7c3e80a902ec7d411ec44f5dc33d56a)